src/main/java/uk/gov/hmcts/idam/userprofilebridge/config/QueueConfig.java

```diff
@@ -51,7 +55,7 @@ public DefaultJmsListenerContainerFactory jmsListenerContainerFactory(Connection
                                                                           ListenerErrorHandler errorHandler,
                                                                           MessageConverter messageConverter) {
         DefaultJmsListenerContainerFactory factory = new DefaultJmsListenerContainerFactory();
-        factory.setPubSubDomain(true);
+        factory.setPubSubDomain(pubSub);
```

Comment:
`setSubPubDomain` is a very poor method name IMO :) Can we name our variable in a way that makes it clearer that `true` means `use topic` and `false` means `use queue`?